src/components/organisms/AdminVenueView/components/Spaces/Spaces.tsx

```diff
@@ -29,16 +29,18 @@ export const Spaces: React.FC<SpacesProps> = ({ venue: space }) => (
     </AdminSidebar>
     <AdminShowcase className="Spaces__map">
       {BACKGROUND_IMG_TEMPLATES.includes(space.template as VenueTemplate) && (
-        <MapPreview
-          isEditing
-          worldId={space.worldId}
-          venueId={space.id}
-          venueName={space.name}
-          mapBackground={space.mapBackgroundImageUrl}
-          rooms={space.rooms ?? []}
-        />
+        <>
+          <MapPreview
+            isEditing
+            worldId={space.worldId}
+            venueId={space.id}
+            venueName={space.name}
+            mapBackground={space.mapBackgroundImageUrl}
+            rooms={space.rooms ?? []}
+          />
+          <PortalsTable space={space} />
+        </>
       )}
-      <PortalsTable space={space} />
```

Comment:
NOTE:
This is not the Create Portal button, but I guess it makes more sense to have the entire component hidden for now (originally the button wasn't supposed to be part of the table component ping @colinhowe )